### PR TITLE
migrating existing position for new interest rate model

### PR DESCRIPTION
**Pull request by RomanHiden**

### CODE REVIEW

**Comment by RomanHiden** [comment]
@eugenesh1 please test

File: scripts/interst-rate-migration/migration/polygon/pause-itokens.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+from ape_safe import ApeSafe
+
+exec(open("./scripts/env/set-matic.py").read())
+exec(open("./scripts/env/common-functions.py").read())
+
+deployer = accounts[0]
+multisig = "0x01F569df8A270eCA78597aFe97D30c65D8a8ca80"
+safe = ApeSafe(multisig)
+
+pauseSignatures=[
```
Comment (RomanHiden) [inline scripts/interst-rate-migration/migration/polygon/pause-itokens.py]: if you do this way than `borrow` also should be paused

File: contracts/modules/PauseGuardian/ProtocolPausableGuardian.sol [hunk 2]
```diff
@@ -20,7 +20,28 @@ contract ProtocolPausableGuardian is State, PausableGuardian {
         _setTarget(this._isPaused.selector, target);
         _setTarget(this.toggleFunctionPause.selector, target);
         _setTarget(this.toggleFunctionUnPause.selector, target);
+        _setTarget(this.pause.selector, target);
+        _setTarget(this.unpause.selector, target);
         _setTarget(this.changeGuardian.selector, target);
         _setTarget(this.getGuardian.selector, target);
     }
+
+    function pause(bytes4 [] calldata sig)
```
Comment (RomanHiden) [inline contracts/modules/PauseGuardian/ProtocolPausableGuardian.sol]: these functions should be in PausableGuardian.sol instead

Comment (ghost) [reply contracts/modules/PauseGuardian/ProtocolPausableGuardian.sol]: Yes, i"ll remove

File: scripts/interst-rate-migration/migration/polygon/deploy-itokens.py [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    print("settings", settings)
+    iTokenProxy = Contract.from_abi("iProxy", address=itoken.address, abi=LoanToken.abi)
+    iToken = Contract.from_abi("loanTokenLogicStandard", iTokenProxy, LoanTokenLogicStandard.abi)
+    print("iToken: ", iToken.name())
+
+    print("setTarget", loanTokenLogicStandard)
+    iTokenProxy.setTarget(loanTokenLogicStandard, {'from': multisig})
+    addToCalldataSet(calldata_set,iTokenProxy.address, iTokenProxy.setTarget.encode_input(loanTokenLogicStandard))
+
+    calldata = settings.initialize.encode_input(token, iToken.name(), iToken.symbol())
```
Comment (RomanHiden) [inline scripts/interst-rate-migration/migration/polygon/deploy-itokens.py:23]: why do we update name? ans symbol

**Review by Drypto13 — APPROVED** [review]

**Pull request merged**